DES: Added SBoxLogic.java

PuravUdayDesai opened the pull request:
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue Number: #1553

PuravUdayDesai committed:
Merge pull request #1 from TesseractCoding/master

Sync Master Repository to Forked Repository

PuravUdayDesai committed:
Added DESController.java

PuravUdayDesai committed:
Moved DESController.java under des/

PuravUdayDesai committed:
Added DESCryptographyLogic.java

PuravUdayDesai committed:
Added DESCryptographyModels.java

PuravUdayDesai committed:
Added InitialPermutationLogic.java

PuravUdayDesai committed:
Added InverseInitialPermutationLogic.java

PuravUdayDesai committed:
Added PBoxLogic.java

PuravUdayDesai committed:
Added SBoxLogic.java

PuravUdayDesai commented:
@HarshCasper please review the pull request once, if any changes/improvements/modifications needed please let me know about the respective.

Sequence Number: 07

Issue Number: #1553 
Pull Request Number:  #2084

HarshCasper commented:
Please have a look at my review here and make the same changes: https://github.com/TesseractCoding/NeoAlgo/pull/2078

PuravUdayDesai committed:
Updated DESController.java

Reduced redundant code for printing the plain text, cipher text and decrypted text.

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #2 from PuravUdayDesai/1553-des-controller

1553 des controller

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #3 from PuravUdayDesai/1553-des-cryptography-logic

1553 des cryptography logic

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #4 from PuravUdayDesai/1553-des-cryptography-models

1553 des cryptography models

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #5 from PuravUdayDesai/1553-des-initial-permutation

1553 des initial permutation

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #6 from PuravUdayDesai/1553-des-inverse-initial-permutation

1553 des inverse initial permutation

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #7 from PuravUdayDesai/1553-des-p-box

1553 des p box

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

HarshCasper commented:
Closing this PR because of no updates.
